cluster-provision/k8s/1.25/manifests/kubeadm_ipv6.conf

```diff
@@ -26,7 +26,11 @@ apiServer:
     enable-admission-plugins: NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota
     feature-gates: AdvancedAuditing=true
     bind-address: "::"
+    admission-control-config-file: /etc/kubernetes/psa.yaml
```

Comment:
I am not sure if we are enabling PSA on providers earlier than 1.26? @acardace - am I right here?

Reply:
we do on 1.25 and it was missing
https://github.com/kubevirt/kubevirtci/pull/915/files#r1052081728

Reply:
Ok thanks @oshoval

Reply:
you mean enabled using `KUBEVIRT_PSA` right ?
`kubeadm.conf` changes do exists on 1.25 as well

Reply:
@oshoval  I guess if a user does the following 
`export KUBEVIRT_SINGLE_STACK=true KUBEVIRT_PROVIDER=k8s-1.24`

They get PSA configuration for a 1.24 cluster?

Reply:
indeed not support for 1.24
moreover the feature determines only if its a single stack IPv6 or dual stack (default)